runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -929,7 +970,7 @@ MM_CopyForwardScheme::reserveMemoryForObject(MM_EnvironmentVLHGC *env, UDATA com
 	if (NULL == result) {
 		region = acquireRegion(env, regionList, compactGroup);
```

Comment:
I know you are not directly changing this methods, but i think it would be nice to rename this method to acquireFreeRegion (or Empty or Idle, or whatever is more appropriate.)